packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -451,15 +452,17 @@ export async function launch({
     ? parseInt(env.END_BLOCK_SPIN_MS, 10)
     : 0;
 
-  const inboundQueueMetrics = makeInboundQueueMetrics(
-    actionQueue.size() + highPriorityQueue.size(),
-  );
-  const { crankScheduler } = exportKernelStats({
+  const initialQueueLengths = /** @type {Record<InboundQueueName, number>} */ ({
+    queued: actionQueue.size(),
+    'high-priority': highPriorityQueue.size(),
+    forced: runThisBlock.size(),
+  });
```

Comment:
I like passing current values instead making metrics just to pass them back.

I wish the `high-priority` phase had a name that better related to the `queued` name.

Reply:
These {'leftover' | 'forced' | 'high-priority' | 'timer' | 'queued' | 'cleanup'} [phase names](https://github.com/Agoric/agoric-sdk/blob/6498ae1ecd1e8ada0eec29d2714e8cf69efbcee9/packages/cosmic-swingset/src/launch-chain.js#L91) are more externally relevant than the internal details to which they correspond, and have been introduced only recently. If we want to change them, there's no better time than now. Do you have any ideas?

Reply:
Renamed "high-priority" → "priority" and "queued" → "inbound" so that each phase is now a single word that looks good embedded in a metric name.